Playbooks/AS-PagerDuty-Integration/azuredeploy.json

```diff
@@ -0,0 +1,392 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "metadata": {
+        "comments": "This playbook is intended to be run from an Microsoft Sentinel Incident. It will generate a PagerDuty Incident with the Microsoft Sentinel Incident information.",
+        "author": "Accelerynt"
+    },
+    "parameters": {
+        "PlaybookName": {
+            "defaultValue": "AS-PagerDuty-Integration",
+            "type": "String"
+        },
+        "IntegrationKey": {
+            "defaultValue": "Enter your PagerDuty Integration Key",
+            "type": "String"

```

Comment:
Mark the data type as securestring as this field contains a secret.

Reply:
Thank you for catching this as well. I will be circling back with the other playbooks we've released here, as no code reviewer has ever made this suggestion. This is resolved.